demo/admin/src/common/ComponentDemo.tsx

```diff
@@ -224,24 +202,18 @@ export function ComponentDemo(): React.ReactElement {
                                 </Field>
                             </FieldContainer>
                             <FieldContainer label="Multiple choice">
-                                <Field name="multiple-choice-1" type="checkbox" fullWidth>
-                                    {(props) => <FormControlLabel label="Option 1" control={<FinalFormCheckbox {...props} />} />}
-                                </Field>
-                                <Field name="multiple-choice-2" type="checkbox" fullWidth>
-                                    {(props) => <FormControlLabel label="Option 2" control={<FinalFormCheckbox {...props} />} />}
-                                </Field>
-                                <Field name="multiple-choice-3" type="checkbox" fullWidth>
-                                    {(props) => <FormControlLabel label="Option 3" control={<FinalFormCheckbox {...props} />} />}
-                                </Field>
+                                <CheckboxField name="multiple-choice-1" type="checkbox" label="Option 1" fullWidth />
+                                <CheckboxField name="multiple-choice-2" type="checkbox" label="Option 2" fullWidth />
+                                <CheckboxField name="multiple-choice-3" type="checkbox" label="Option 3" fullWidth />
                             </FieldContainer>
 
                             <Field name="switch" label="Switch with label">
                                 {(props) => <FormControlLabel label={null} control={<FinalFormSwitch {...props} />} />}
                             </Field>
 
-                            <Field name="switch">
-                                {(props) => <FormControlLabel label="Switch with inline label" control={<FinalFormSwitch {...props} />} />}
-                            </Field>
+                            <SwitchField name="switch" fieldLabel="Switch with inline label" />
```

Comment:
Shouldn't the `SwitchField` with the `fieldLabel` replace the `Field` above and have that field's label as well?